web/src/pages/Admin/tools/makeData.jsx

```diff
@@ -0,0 +1,40 @@
+import namor from 'namor';
```

Comment:
This file should be removed - it's no longer used since we're sourcing data from gql